nautobot_ssot/integrations/aristacv/diffsync/adapters/cloudvision.py

```diff
@@ -55,7 +55,9 @@ def load_devices(self):
                 self.add(new_cvp)
             except ObjectAlreadyExists as err:
                 self.job.logger.warning(f"Error attempting to add CloudVision device. {err}")
-        for dev in cloudvision.get_devices(client=self.conn.comm_channel):
+
+        for index, dev in enumerate(cloudvision.get_devices(client=self.conn.comm_channel), start=1):
+            self.job.logger.info(f"Loading {index}° device")
```

Comment:
Do we want the `index` or the `dev`here?

Reply:
Hi @qduk ,
In my architecture, i have 150 devices and during the normal execution i would to know the progress. I insert the index to have a rapid view but it's a good idea to add the device name too.

Reply:
Ah I see. Yes, I think having the device name in the log message may be beneficial.